update code of conduct to include weapons policy (#34)
djangocon/2024.djangocon.us: _posts/2024-04-05-call-for-2024-volunteers.md

---
author: Abigail Afi Gbadago
category: General
date: 2024-04-05 12:00:00-07:00
layout: post
title: "Call for Volunteers: DjangoCon US 2024"
---

> As our program chair, Drew Winstel, mentioned in his opening remarks today, we would love to have more volunteers for DjangoCon US 2023. Expect to hear details on dates and venues within the next month or so.

Reviewer: ```suggestion
Organizing DjangoCon US 2024 has begun! We need organizers to help make DjangoCon US 2024 the best it can be.
```

Since those opening remarks are now a year and a half old, we probably can't say 'today' 😆

We have some [documentation about the various roles](https://djangocon.github.io/djangocon-us-docs/), but here's a quick overview of the teams at DjangoCon US.

The DEFNA board will not leave you stranded; we have a lot of documentation to help you on your way, including rough checklists of tasks needed for each team.

Ideally, all committees will have a chair and co-chair to allow for one or the other to take vacations, handle day job duties, and so forth. This list is by no means comprehensive, but it should hopefully give you an idea of what each committee is responsible for.

## Conference Leadership

> The conference chair and co-chair lead the conference, including coordinating between teams and making executive decisions (including approaching the DEFNA board when applicable). Expect to be in meetings with the venue and online conference provider, weekly organizer check-ins, monthly reports to the DEFNA board, and lots of questions on Slack. You'll also lead the volunteer kickoff meetings in early 2023 (ideally mid-to-late January) and recruit organizers as you see fit. You'll need to recommend an online conference provider to the DEFNA board (might be part of a sponsorship deal).

Reviewer 2: ```suggestion
The conference chair and co-chair lead the conference, including coordinating between teams and making executive decisions (including approaching the DEFNA board when applicable). Expect to be in meetings with the venue and online conference provider, weekly organizer check-ins, monthly reports to the DEFNA board, and lots of questions on Slack. You'll also lead the volunteer kickoff meetings in early 2024 (ideally mid-to-late January) and recruit organizers as you see fit. You'll need to recommend an online conference provider to the DEFNA board (might be part of a sponsorship deal).
```

At the conference, you'll be making speeches at the start of each day and the end-of-conference wrap-up in addition to coordinating setup, teardown, and everything in between.

These two roles have the highest time commitment and the most visibility. If you want to help people get things done and get up on stage, these are the roles for you.

## Program Committee

Size: 2-4

The program committee is responsible for making sure that the schedule of tutorials and talks is the best it can be.

### Pre-conference responsibilities

> - Start the call for proposals process (ideally opening in February or March 2023)

Reviewer: ```suggestion
- Start the call for proposals process (opened in March 2024, closes April 24)
```

### At-conference responsibilities

- Work with the venue on any work needed for social events

## Venue Committee

Size: 1-2

The venue committee is responsible for coordinating plans between the other organizing teams and the venue(s).

### Pre-conference responsibilities

- Work with the event venue to select and lay out talk, tutorial, and sprint rooms
- Lay out the extra rooms needed for the conference (lactation room, quiet room, organizers room, speaker green room, lunch/break areas)
- Work with event venue and DEFNA to select catering menus (and providers if we're not contractually obligated to use one)
- Work with the hotel to coordinate room blocks
- Work with the venue to plan logistics (sponsor gear shipping, signage, etc.)
- Allocate reward rooms in coordination with the DEFNA board
- Order supplies for restroom stocking

### At-conference responsibilities

- Review room guest list against ticket list to claim reward nights
- Handle any last-minute requests from vendors/sponsors/organizers/etc.
- Notify venue staff of any needs (bathroom cleanliness, meals to accommodate special diets, etc.)

## Online Committee

Size: 2-3

### During-conference responsibilities

> - Set up online tutorials

Reviewer 2: ```suggestion
```

Our tutorials are back in person this year.

### Pre-conference responsibilities

- Design and order attendee t-shirts (with sponsor logos)
- Design and order swag bags (with sponsor logos)
- Design and order speaker gifts
- Design and order DCUS-branded swag (stickers/magnets/etc.)
- Order masks and COVID tests (if needed)
- Order lanyards, badges, and badge flags

### At-conference responsibilities

- Recruit volunteers to stuff swag bags on Sunday afternoon
- Get T-shirts and swag bags to the registration desk
- Get speaker gifts to the program team for distribution

## Volunteer Committee

Size: 2

The volunteer committee is responsible for cultivating new volunteers, who can become organizers in future years.

### Pre-conference responsibilities

- Determine what roles need on-site volunteer support (session chairs, session managers, registration desk, etc.)
- Set up sign-up for on-site volunteers and online volunteers (moderators/cheerleaders)
- Draft announcements for calls for volunteers

### At-conference responsibilities

- Tell volunteers where to go
- Give volunteers tools to succeed

## Communications Committee

Size: 2-4

The communications committee is responsible for getting the word out about DCUS.

> Have you heard the expression "never tweet" (usually said on Twitter)? This is the opposite of that. Always tweet.

Reviewer: ```suggestion
This team manages the various DCUS social network feeds, including Twitter, Mastodon, Bluesky, Instagram, LinkedIn, and more. This team also edits and posts blogs on the DCUS main page and coordinates bulk emails to ticketholders to remind them about the conference as it approaches.
```

### Pre-conference responsibilities

- Publish announcements and reminders on social media, the conference blog, and more (Django News, Python Community News, etc.)
- Send a copy to the Django Software Foundation to get major news events on the Django blog
- Review, edit, and suggest copy for the other teams' announcements
- Tweet. Lots of tweeting.
- Select an "official emoji" for the conference (typically suggested by the design package for the year's website)
- Boost and retweet speakers and organizers talking about the conference
- Schedule tweets announcing talks and events

### At-conference responsibilities

> - Pay attention to the `#djangocon` topic on social media and mentions to `@djangocon` on Twitter

Reviewer: ```suggestion
- Pay attention to the `#djangocon` topic on social media and mentions to `@djangocon` on various social networks
```

### Pre-conference responsibilities

- Update static pages for CFP status
- Update static pages for venue information
- Review pull requests from other teams
- Add sponsors as directed by the sponsorship team

## Automation Team

This team is more free-flowing and writes scripts to make teams' lives easier. Responsibilities vary based on the year, and if you have ideas for special projects for the future, have at it!

## Sprints Team

Size: 2-4

### Pre-conference responsibilities

- Write blog posts announcing sprints
- Invite people to launch special sessions (e.g. Carlton's ["Getting Started Contributing to Django"](https://2022.djangocon.us/sprints/getting-started-contributing-to-django/))
- Ensure we have signage and power equipment

### At-conference responsibilities

- Coordinate help desk
- Direct people to lunches and breaks

## Wrap-up

> To sign up for any of the teams (or just say "Put me to work"), send an email to hello@djangocon.us and we'll put you on the list for the 2023 kickoff meeting. We would love to have you! You do not have to have previous organizing experience (although we might prefer it for some of the leadership roles).

Reviewer: ```suggestion
To sign up for any of the teams (or just say "Put me to work"), send an email to hello@djangocon.us and we'll add you to our organizer Slack instance and invite you to our fortnightly organizer catch-up meetings. We would love to have you! You do not have to have previous organizing experience (although we might prefer it for some of the leadership roles).
```
